lib/graphql/schema/enum.rb

```diff
@@ -67,6 +67,7 @@ class << self
         def value(*args, **kwargs, &block)
           kwargs[:owner] = self
           value = enum_value_class.new(*args, **kwargs, &block)
+          singleton_class.define_method(value.graphql_name.downcase) { value.graphql_name }
```

Comment:
Implementation-wise, calling `define_method` with a block will result in the `binding` of this method staying in memory for the lifetime of the application. I'd like to avoid that memory "bloat", and we could probably do it with something like 

```ruby 
instance_eval("def #{value.graphql_name.downcase}; #{value.graphql_name.inspect}; end;")
```

It would create the method that returns the string, but without retaining a reference of the binding.